modules/page-memory/src/main/java/org/apache/ignite/internal/pagememory/persistence/checkpoint/CheckpointManager.java

```diff
@@ -191,6 +199,13 @@ public CheckpointProgress forceCheckpoint(String reason) {
         return checkpointer.scheduleCheckpoint(0, reason);
     }
 
+    /**
+     * Returns progress of current checkpoint, last finished one or {@code null}, if checkpoint has never started.
+     */
+    public @Nullable CheckpointProgress currentProgress() {
```

Comment:
Now I see an answer to my previous question. Can you please rename it to "lastCheckpointProgress" so that no one would have the same question?

Reply:
Fix it